docs/manuals/deployment/docker/development.md

```diff
@@ -23,12 +21,12 @@ COPY . /app
 RUN pip install -r requirements.txt
 
 # Start up command
-CMD python my-app.py
+CMD python main.py
 ```
 
 !!! Note
 
-    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to add the following commands
+    If you are using an `SQLDataNode^` based on Microsoft SQL Server, you need to add the following commands
```

Comment:
So the doc also works for future sql storage for enterprise.
```suggestion
    If you are using an SQL database based on Microsoft SQL Server, you need to add the following commands
```

What do you think?